applications/GeoMechanicsApplication/custom_conditions/T_normal_flux_condition.h

```diff
@@ -27,11 +25,9 @@ template <unsigned int TDim, unsigned int TNumNodes>
 class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoTNormalFluxCondition
     : public GeoTCondition<TDim, TNumNodes> {
 public:
-    using NodeType = Node;
```

Comment:
Also here, I don't see the added value of the `NodeType` alias. I prefer to have a single type (`Node`) rather than it being aliased to `NodeType` in many classes. So I'm in favor of removing this alias.